.github/workflows/mac-test.yml

```diff
@@ -25,6 +25,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
+    
+    - name: Install FFTW
+      run: brew install fftw
```

Comment:
Maybe you already tried, but I would install FFTW from the `conda_deps` in `tox.ini`.